src/resources/sfml_loaders.h

```diff
@@ -37,25 +38,38 @@ el::ResourceLoader<ResourceType> makeResourceLoader(Func func,
   return el::ResourceLoader<ResourceType>(loader, key);
 }
 
-}  // namespace detail
+} // namespace detail
 
 template <typename ResourceType>
 // ResourceType: The type of resource we load.
-el::ResourceLoader<ResourceType> fromFile(const std::string& filename) {
-  return detail::makeResourceLoader<ResourceType>([=](ResourceType& resource) {
+el::ResourceLoader<ResourceType>
+fromFile(const nu::FilePath::StringType &filename) {
+  return detail::makeResourceLoader<ResourceType>([=](ResourceType &resource) {
+#if OS(POSIX)
     return resource.loadFromFile(filename);
+#else
```

Comment:
We should change ca::Texture to take a nu::FilePath as well, to avoid there everywhere.